lib/lambda/search.ts

```diff
@@ -48,25 +51,35 @@ export const getSearchData = async (event: APIGatewayEvent) => {
       },
     });
 
+    // Pagination parameters with defaults
     query.from = query.from || 0;
     query.size = query.size || 100;
 
     const results = await os.search(domain, index, query);
 
-    for (let i = 0; i < results?.hits?.hits?.length; i++) {
-      if (results.hits.hits[i]._source?.appkParent) {
-        const children = await getAppkChildren(results.hits.hits[i]._id);
-        if (children?.hits?.hits?.length > 0) {
-          results.hits.hits[i]._source.appkChildren = children.hits.hits;
-        }
-      }
-    }
+    // Fetch children for appkParent documents
+    const childrenPromises =
+      results?.hits?.hits
```

Comment:
This breaks

Reply:
Lol sorry, it was more of a "this breaks all things" kind of thing.  I think.  Something is causing an issue with certain states to be reloaded over and over again.  I was trying to narrow down the issue but was unable to locate it.  The issue is some file that has the word "search" in it.  A change done something in either a hook or logic is causing the issue.  I would recommend breaking the PR down into smaller parts for individual approval because its pretty difficult to find the underlying issue.